tekton/pipeline_run.go

```diff
@@ -140,5 +140,15 @@ func (r *ReleasePipelineRun) WithReleaseStrategy(strategy *v1alpha1.ReleaseStrat
 		})
 	}
 
+	r.WithServiceAccount(strategy.Spec.ServiceAccount)
+
+	return r
+}
+
+// WithServiceAccount adds a reference to the service account to be used to gain elevated privileges during the
+// execution of the different Pipeline tasks.
+func (r *ReleasePipelineRun) WithServiceAccount(serviceAccount string) *ReleasePipelineRun {
+	r.Spec.ServiceAccountName = serviceAccount
```

Comment:
I am undecided if we should just have this line be in place of line 143 or not. I'll think on it

Reply:
Having it this way will allow us to set a ServiceAccount independently of the ReleaseStrategy. Not like we are going to need it, but it's cleaner.

Reply:
I don't follow. line 143 just calls line 151. What makes it independent/dependent of the ReleaseStrategy?

Reply:
You can call WithServiceAccount without needing to call WithReleaseStrategy.

Reply:
I doubt we do that, but fair